cmd/coverage/main_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"flag"
+	"testing"
+)
+
+func Test_Main(t *testing.T) {
+	pathToDoc := "/home/tbex/go/src/github.com/terraform-providers/terraform-provider-vault/testdata/openapi.json"
```

Comment:
May want to consider using a path in `/tmp` or something that isn't unique to your local machine.

Reply:
Weird that showed up for you, it was that way during my first commit but not the second. Should be currently resolved on the branch to be a relative directory.